assets/src/adaptivity/scripting.ts

```diff
@@ -612,26 +612,26 @@ export const templatizeText = (
   useFormattedText = true,
   key?: string,
 ): string => {
-  let shouldEvaluateExpression = true;
-  try {
-    //To improve the performance, when a lesson is opened in authoring, we generate a list of variables that contains expression and needs evaluation
-    // we stored them in conditionsNeedEvaluation in activity.content.custom.conditionsNeedEvaluation. When this function is called
-    // we only process variables that is present in conditionsNeedEvaluation array and ignore others.
-    const conditionsNeedEvaluations = Object.keys(locals)?.length
-      ? locals['session.conditionsNeedEvaluation']
-      : getValue('session.conditionsNeedEvaluation', defaultGlobalEnv);
-
-    // if they key is not passed then it means that this function was called from the janu-text component so this logic will not apply
-    // we need to process it with the old behaviour
-    if (conditionsNeedEvaluations?.length && key) {
-      shouldEvaluateExpression = conditionsNeedEvaluations.includes(key);
-    }
-    if (!shouldEvaluateExpression) {
-      return text;
-    }
-  } catch (er) {
-    console.warn('Error at templatizeText for key', { key });
-  }
+  //const shouldEvaluateExpression = true;
```

Comment:
Commenting out the entire block here so that as usual `templatizeText `function always try to evaluate the text irrespective of whether it has expression or not.